### PR TITLE
Updating Assorted Documentation

### PR DESCRIPTION
**Pull request by zklosko**
Hey all, I updated a few pages on the Docs site to add clarity to the install process, explain an error that can happen when adding a new webstream, and adding instructions for changing the radio page background. This is my first time submitting a PR for the project, so please let me know if I need to change anything. Thanks!

### CODE REVIEW

**Comment by Robbt** [comment]
This looks good to me, thanks for contributing, we should be able to merge this no problem.

**Comment by zklosko** [comment]
Thanks @Robbt. I just set up LibreTime on a headless Ubuntu Server for WRIR-LP in Richmond, VA, to help our volunteers out during the pandemic. Looking through my install notes, I've got plenty more to add; is it okay to let this sit for a few days so I can get more notes in?

**Comment by Robbt** [comment]
Totally, I'll add [wip] to the title, when you are done you can just delete that.

File: docs/manual/webstreams/index.md [hunk 1]
```diff
@@ -8,3 +8,5 @@ To add a web stream, click the **New** button on the right side of the Library p
 ![](static/Screenshot516-New_remote_webstream.png)
 
 The **Stream URL** setting must include the *port number* (such as 8000) and *mount point* (such as remote\_stream) of the remote stream, in addition to the streaming server name. A **Default Length** for the remote stream can also be set. If the stream is added at the end of a show which becomes overbooked as a result, it will be faded out when the show ends.
+
+Note: LibreTime checks the remote webstream's status upon editing stream settings, so an offline stream will result in an error. There are many tools such as [BUTT](https://danielnoethen.de/butt/) and [MIXXX](https://www.mixxx.org) that can be used to send a test stream to LibreTime can save it; read more [here](https://libretime.org/manual/live-shows-with-mixxx/).
```
Comment (paddatrapper) [inline docs/manual/webstreams/index.md:12]: This link should be an relative reference, not to the libretime.org site

Comment (zklosko) [reply docs/manual/webstreams/index.md]: Fixed.

**Comment by zklosko** [comment]
I think it might be good for now. Removed [wip] from the title.

**Comment by paddatrapper** [comment]
LGTM, thanks!

**Pull request merged**